NAMESPACE

```diff
@@ -3,6 +3,8 @@
 export(add_methods_step)
 export(clear_methods)
 export(convert_iso_to_eml)
+export(create_dummy_attributes_dataframe)
```

Comment:
Do you want these to be exported? If they're just used for examples or tests, consider:

- Just removing the `@export`
- Refactor examples not to need a helper like this by creating what's needed in the example
- Load what you need from disk by storing either a CSV or an RData in the `inst` folder). There are some examples already in there and this is pretty common practice.

Reply:
Hmmm. These two helpers that I added were actually code snippets of Dominic's, and others in the datateam have been copying/pasting this bit to create test attribute tables and data tables. I figured since I needed that in my test I'd go ahead and make the functionality public so that others on the team could use it, but perhaps this would be better in datamgmt? Also happy to just add this as a CSV in `inst` if you'd prefer that.

Reply:
Oh, that's great. Please do keep these helpers around then. That's great to hear it's useful.